suministrospr/suministros/templates/suministros/suministro_detail.html

```diff
@@ -17,7 +17,7 @@
 <div class="flex justify-center">
   <div class="flex flex-row text-sm text-gray-600">
     <div class="ml-2">
-      <a href="{% url 'suministro-municipio-list' object.municipality|slugify %}">{{ object.get_municipality_display }}</a>
+      <a href="{% url 'suministro-municipio-list' object.municipality|slugify %}">{{ object.municipality.get_name_display }}</a>
```

Comment:
@jonatns Thank you!

Although technically correct, can we change this to:

```suggestion
      <a href="{% url 'suministro-municipio-list' object.municipality.slug %}">{{ object.municipality.name }}</a>
```

Reply:
Updated! @jpadilla